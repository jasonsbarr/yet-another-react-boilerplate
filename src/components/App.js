import React from "react";

const App = () => <div>React App - App component</div>;

export default App;
