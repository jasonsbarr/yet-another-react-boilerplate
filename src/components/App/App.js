import React from "react";
import { Routes } from "Routes";
import styles from "./App.module.scss";

const App = () => (
  <div className={styles.App}>
    <Routes />
  </div>
);

export default App;
