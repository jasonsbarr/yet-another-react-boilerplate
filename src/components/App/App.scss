.App {
  margin: 0 auto;
  max-width: 1080px;
  width: 80%;
}
