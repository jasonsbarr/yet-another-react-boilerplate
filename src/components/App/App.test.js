import React from "react";
import { BrowserRouter } from "react-router-dom";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import App from "./App";

afterAll(cleanup);

describe("App component tests", () => {
  test("should render Home view from root route", () => {
    const app = render(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );

    expect(app.asFragment()).toMatchSnapshot();
  });
});
