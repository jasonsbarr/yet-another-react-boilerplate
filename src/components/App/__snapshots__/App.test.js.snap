// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests should render Home view from root route 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      React App - home
    </div>
  </div>
</DocumentFragment>
`;
